src/test/java/de/qabel/desktop/ui/LayoutControllerTest.java

```diff
@@ -28,17 +33,30 @@ public void getQuotaStateFailuresThenHidesQuotaBars() {
     @Test
     public void testFillQuotaInformation() {
         Locale.setDefault(new Locale("te", "ST"));
+        ((BoxClientStub) boxClient).quotaState = setQuota();
```

Comment:
you can probably change the type to `BoxClientStub` at it's declaration (AbstractControllerTest?). It's ok if the test depends on the stub it uses (because its dependend anyways here)